### PR TITLE
add tests for the examples

### LINKED ISSUE #98: Create a unit test to run all the examples
There are examples scripts all over the repository which are broken after most refactors. Write a unit test to run the example script since these are scripts they is no return value to validate so we are just running them to ensure they don't fail. If there is a way to validate the result from the script, this is also okay.

### PR DESCRIPTION
**Pull request by RabiyaF**
fixes #98 

Test have been added for the six RAT examples. They are run to verify if examples scripts can be run. Various properties of the problem variable are also verified in tests.

### CODE REVIEW

File: tests/testExamples.m [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+            4,...
+            2,...
+            2}; 
+        exampleLayerCount = {5,...
+            0,...
+            0,...
+            4,...
+            4,...
+            0};
+        exampleGeometry = {'air/substrate',...
```
Comment (StephenNneji) [inline tests/testExamples.m:34]: Use enums instead i.e `geometryOptions.AirSubstrate.value`

File: tests/twoContrastExample.mat
Comment (StephenNneji) [inline tests/twoContrastExample.mat:1]: Is this generated by the test or do you use this? if it is generated by the test, please add a tear down function to delete this

**Review by StephenNneji — APPROVED** [review]
Please see comments

File: tests/testExamples.m [hunk 2]
```diff
@@ -0,0 +1,114 @@
+classdef testExamples < matlab.unittest.TestCase
+
+    properties (TestParameter)
+        exampleFile = {'orsoDSPCStandardLayers',...
+            'DPPCCustomXYScript',...
+            'testCppCustomLayers',...
+            'DSPCMonolayerStandardLayers',...
+            'DSPCMonolayerTwoContrasts',...
+            'DSPCMonolayerCustomLayers'};
```
[the remark this answers is not in the document]
Comment (RabiyaF) [reply tests/testExamples.m]: I have removed the DREAM test for now. I tested all six of the test yesterday and they were all working locally. However, 2 tests are not working anymore. I am getting the following error. 

image

[the remark this answers is not in the document]
Comment (DrPaulSharp) [reply tests/testExamples.m]: Ahh, good. The PR has rebuilt so we're all fine.

[the remark this answers is not in the document]
Comment (RabiyaF) [reply tests/testExamples.m]: Thank you so much.

**Pull request merged**